config.json

```diff
@@ -3,26 +3,28 @@
   "documentation": "https://docs.docker.com/engine/extend/plugins/",
   "entrypoint": [ "/docker-volume-linode" ],
   "env": [
-    { "name": "LOG_LEVEL",  "settable": [ "value" ], "value": "INFO" },
```

Comment:
Changed parameters to match command line arguments.  This makes it possible to use the same name when calling plugin install:
docker plugin install linode/linode-volume-linode linode-token=abcde1234